# Vehicle state

Header header
string name
geometry_msgs/Point position
Euler orientation
Velocity ground_velocity